src/assets/Standards/ERC20Standard.ts

```diff
@@ -0,0 +1,78 @@
+import { BN } from 'ethereumjs-util';
+import { ERC20 } from '../../constants';
+
+export class ERC20Standard {
+  /**
+   * Get balance or count for current account on specific asset contract.
+   *
+   * @param contract - Asset ERC20 contract.
+   * @param selectedAddress - Current account public address.
+   * @returns Promise resolving to BN object containing balance for current account on specific asset contract.
+   */
+  async getBalanceOf(contract: any, selectedAddress: string): Promise<BN> {
```

Comment:
Is there a type we can use other than `any` here? Maybe something like:

``` typescript
{
  balanceOf: (
    address: `0x${string}`;
    callback: (error: Error | null, result: BN | null) => void;
  ) => void;
}
```

Same thing for the other instances of `contract` in this file.

Reply:
This pattern has actually been changed now, so that each of these methods receives the contract address and instantiates the contract itself. So this issue is overcome by events. 😄